Resources/Prototypes/_CD/Recipes/Construction/Graphs/Furniture/beds.yml

```diff
@@ -0,0 +1,62 @@
+- type: constructionGraph
+  id: bedNF
+  start: start
+  graph:
+    - node: start
+      actions:
+        - !type:DestroyEntity {}
+      edges:
+        - to: mattress
+          completed:
+            - !type:SnapToGrid { }
+          steps:
+            - material: Cloth
+              amount: 2
+              doAfter: 2
+        - to: PsychBed
+          completed:
+            - !type:SnapToGrid { }
+          steps:
+            - material: WoodPlank
+              amount: 4
+            - material: Cloth
+              amount: 4
+              doAfter: 2
+    - node: mattress
+      entity: Mattress
+      edges:
+        - to: start
+          completed:
+            - !type:SpawnPrototype
+              prototype: MaterialCloth1
+              amount: 2
+          steps:
+            - tool: Screwing
+              doAfter: 1
+    - node: PsychBed
+      entity: PsychBed
+      edges:
+        - to: start
+          completed:
+            - !type:SpawnPrototype
+              prototype: MaterialWoodPlank1
+              amount: 4
+            - !type:SpawnPrototype
+              prototype: MaterialCloth1
+              amount: 4
+          steps:
+            - tool: Screwing
+              doAfter: 1
```

Comment:
I also know I need to get rid of this, since it's all frontierstation stuff, but if I do it crashes when I try and make the nest. What should I do here?